spec/integration/api/streams_controller_spec.rb

```diff
@@ -0,0 +1,490 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Api::V1::StreamsController do
+  let(:auth_read_only) {
+    FactoryGirl.create(
+      :auth_with_default_scopes,
+      scopes: %w[openid public:read private:read contacts:read tags:read],
+      user: FactoryGirl.create(:user, profile: FactoryGirl.create(:profile_with_image_url))
+    )
+  }
+
+  let(:auth_public_only_tags) {
+    FactoryGirl.create(
+      :auth_with_default_scopes,
+      scopes: %w[openid public:read tags:read],
+      user: FactoryGirl.create(:user, profile: FactoryGirl.create(:profile_with_image_url))
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.